docs/tutorials/0_preparation.md

```diff
@@ -0,0 +1,40 @@
+# AutowareArchitectureProposal Tutorials
+
+The following tutorials were originally created for a workshop given at the 2020 Arm DevSummit event, and have been modified for inclusion in the AutowareArchitectureProposal repository. If you have any comments or feedback regarding how these tutorials can be improved, then please feel free to create an issue and @ the original author or the last person who edited the tutorial file.
+
+- [Tutorial 1: RViz basics and replaying a rosbag](1_rosbag.md)
+- [Tutorial 2: Localization using NDT Scan Matching](2_localization.md)
+- [Tutorial 3: Simulation testing with dummy objects](3_simulation.md)
+
+We assume that you've [installed and set up the AutowareArchitectureProposal already](https://github.com/tier4/AutowareArchitectureProposal.proj#autoware-setup).
+
+## Simulation in Autoware
+
+Autoware provides two types of simulation:
+- rosbag-based simulation that can be used for testing/validation of the `Sensing`, `Localization` and `Perception` stacks (this was covered in the first two tutorials)
+- The Planning Simulator tool which is mainly used for testing/validation of `Planning` stack by simulating traffic rules, interactions with dynamic objects and control commands to the ego vehicle (covered in this tutorial)
```

Comment:
@LalithVipulananthan `(covered in this tutorial)` -> `(covered in the third tutorial)` ?

Reply:
Thank you! I've updated the file accordingly.